docs/source/index.md

```diff
@@ -1,17 +1,17 @@
-```{image} https://raw.githubusercontent.com/kedro-org/kedro/main/static/img/kedro_banner.png
-:alt: Kedro logo
-:class: kedro-logo
-```
-
 # Welcome to Kedro-Viz documentation!
```

Comment:
I'll take a look at the paragraph symbol and resolve in the PR I'm working on which is adjusting content links etc. 

I don't think we should aim for consistency with this header because (a) it sux and (b) things are going to diverge now anyway as the Viz docs are more independent. For something like this, divergence is fine. If it was a more stylistic thing, like Viz docs started using US spelling 😛 or different heading capitalisation, then I'd say we should resolve.

Reply:
This is now resolved in #1628